index.html

```diff
@@ -0,0 +1,190 @@
+            </div>
+        </div>
+        <div class="row">
+            <div class="col">
+                <h2><a href="detail.html" class="full-d">See the Whole Program</a></h2>
+            </div>
+        </div>
+    </div>
+    </section>
+    <section class="program mobile">
+    <div class="container">
+        <div class="row">
+            <div class="col">
+                <h1>Main Program</h1>
+            </div>
+        </div>
+        <div class="col card-container">
+            <div class="p-card">
+                <img class="lecture-img"src="./images/lecture-card1.svg" alt="lecture">
+                <h2>Lecture</h2>
+                <p>Lectures from the distinguished investors.</p>
+            </div>
+            <div class="p-card">
+                <img src="./images/network-card2.svg" alt="network">
+                <h2>Network</h2>
+                <p>Trading with networking</p>
+            </div>
+            <div class="p-card">
+                <img  src="./images/ai-card3.svg" alt="AI">
+                <h2>A.I.</h2>
+                <p>Importance of AI in trading</p>
+            </div>
+            <div class="p-card">
+                <img src="./images/forum-card4.svg" alt="forum">
+                <h2>Forum</h2>
+                <p>Question and Answer session</p>
+            </div>
+            <div class="p-card">
+                <img src="./images/workshop-card5.svg" alt="workshop">
+                <h2>Workshop</h2>
+                <p>Fundamental trading class for enthusiasts</p>
+            </div>
+        </div>
+    </div>
+    </section>
+
+    <!-- Speakers section -->
+    <section class="speakers mobile" id="speaker-mobile">
+                                        
+    </section>
+    <section class="speakers desktop" id="speaker-desktop">
+        
+    </section>
+
+
+    <!-- Partners section -->
+    <section class="partners desktop" id="partners">
+        
+        <h1>Partners</h1>
+
+        <div class="sponsors">
+            <img src="./images/airbnb 1.png">
+            <img src="./images/amz.png">
+            <img src="./images/mozilla 1.png">
+            <img src="./images/coke-removebg-preview.png">
+            <img src="./images/tesla.png">
+        </div>
+    </section>
+
+<footer>
+  <div class="footer-container">
+    <div class="logo-content">
+      <img src="./images/BeatTheMarket-removebg-preview.png" alt="Logo">
+    </div>
+    <div class="footer-content">
+      <h2>&copy; 2023 BEATtheMARKET. Some rights reserved</h2>
+      <p></p>
+    </div>
+  </div>
+</footer>
+
+    <script src="./javascripts/detail.js"></script>
+    <script src="./javascripts/menu-button.js"></script>
+    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+</body>
+</html>
```

Comment:
Kindly implement the following changes for the mobile view of your homepage :

1.  for your mobile menu, kindly fix the position issue as the background color doesn't extend to the end, fix the margin as the items are to be centered and also use the color guideline for the project here 😄 
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/3390f644-32d9-4a72-ba0d-f19d795248b6)

3. Kindly add this button in your mobile version 

![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/fb7c0b99-0c21-4db1-94f9-8fd8595f086c)

3. Please implement the speakers section to be similar to the design given

Your design :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/08f684fa-6a44-44fc-afda-de6435c7f2ad)

Design given :
![image](https://github.com/AliRezaBmeDu/conference-capstone/assets/20537712/b1e84d3b-c6ce-463c-acf5-9018debf3455)